src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
renderV2 => toJson in https://github.com/telefonicaid/fiware-orion/commit/0bc1f0199f2bc6055a2c24ae3ab24a71a422674c